website/scripts/should-deploy-docs.js

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env node
+
+const {execSync} = require('child_process');
+
+const {TRAVIS_COMMIT_RANGE = '', TRAVIS_COMMIT_MESSAGE = ''} = process.env;
+
+console.log(`TRAVIS_COMMIT_RANGE=${TRAVIS_COMMIT_RANGE}`);
+console.log(`TRAVIS_COMMIT_MESSAGE=${TRAVIS_COMMIT_MESSAGE}`);
+
+let commitMessages;
+
+if (TRAVIS_COMMIT_RANGE) {
+  // see https://github.com/travis-ci/travis-ci/issues/4596
+  const COMMIT_RANGE = TRAVIS_COMMIT_RANGE.replace('...', '..');
+  const BASE_COMMIT = COMMIT_RANGE.split('..')[0];
+
+  const baseCommitType = execSync(`git cat-file -t ${BASE_COMMIT}`)
```

Comment:
I need to add try-catch for this or add `|| true` at the end of command, since it will error out anyway...